bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -30,6 +31,8 @@ namespace Bit.Sso.Controllers
 {
     public class AccountController : Controller
     {
+        #region Members
```

Comment:
Not sure if using regions like this is a style we have adopted in the codebase?

Reply:
![image](https://user-images.githubusercontent.com/42616259/163226080-c82acd3c-6caa-4cdd-b668-2f10eb690d9c.png)
I can remove them but it is nice to open a code file and see it nice and tidy like this, as well as things grouped by accessibility. I figured I'd try to offer the standard with this change but understand it can be too impactful for now ...